src/Explorer/Tree/Collection.ts

```diff
@@ -1331,6 +1331,20 @@ export default class Collection implements ViewModels.Collection {
     return this.container.findDatabaseWithId(this.databaseId);
   }
 
+  public async loadAutopilotOfferWithRetry(): Promise<DataModels.Offer> {
+    let retryCount = 0;
+    while (retryCount < Constants.LoadOfferRetry.MaxRetries) {
+      if (this.offer().content.offerAutopilotSettings) {
```

Comment:
We do this retry since the RP API for updateoffer doesnt show the right data in its response?

if so, does the RP API owner know about this?

Reply:
Nope, we need to track it in the uber excel sheet.

Reply:
Can you please respond to the email thread we have for mongo editor that you see this issue.  

---
In reply to: [521598445](https://github.com/Azure/cosmos-explorer/pull/318#discussion_r521598445) [](ancestors = 521598445)